File: app/src/main/java/com/example/ikastyle/Entity/ShoesGear.java
package com.example.ikastyle.Entity;

import androidx.annotation.NonNull;
import androidx.room.ColumnInfo;
import androidx.room.Entity;

@Entity(tableName = "MAST_SHOES_GEAR", primaryKeys = {"id", "language_code"})
public class ShoesGear {
    @ColumnInfo(name = "id")
    public int id;

    @ColumnInfo(name = "language_code")
    public int languageCode;

    @ColumnInfo(name = "name")
    @NonNull
    public String name;
}

File: app/src/main/java/com/splatool/ikastyle/entity/ShoesGear.kt
package com.splatool.ikastyle.entity

import androidx.annotation.NonNull
import androidx.room.*

@Entity(tableName = "MAST_SHOES_GEAR", primaryKeys = ["id", "language_code"])
data class ShoesGear (
    @ColumnInfo(name = "id") val id : Int,
    @ColumnInfo(name = "language_code") val languageCode : Int,
    @ColumnInfo(name = "name") @NonNull val name: String
)